DOCS/man/options.rst

```diff
@@ -5760,6 +5760,10 @@ them.
         from Windows 10. Thus on older systems it will only automatically
         utilize the rgba8 output format.
 
+    .. note::
+
+        For ``--vo=gpu-next``, this is ignored for HDR or wide gamut color spaces.
```

Comment:
I think this is ambiguous what does "color space" mean in this context. I understand how it works, but I don't expect anyone to understand what this line mean without knowledge of gpu-next internals.